web-frontend/package.json

```
@@ -17,7 +17,7 @@
     "prepare": "cd .. && husky install web-frontend/.husky"
   },
   "dependencies": {
-    "@massalabs/react-ui-kit": "^0.0.2-dev",
+    "@massalabs/react-ui-kit": "^0.0.4-dev.20230510141954",
```

we require version ^0.0.4-dev. please run npm i after applying this

```suggestion
    "@massalabs/react-ui-kit": "^0.0.4-dev",
```

---

Done